python/ray/tests/BUILD

```diff
@@ -679,7 +679,6 @@ py_test(
 # no way to filter by both flaky and client mode tests in bazel.
 py_test_module_list(
   files = [
-    "test_actor.py",
```

Comment:
Actually I saw that bazel "large" timeout is 900s per https://bazel.build/reference/test-encyclopedia?